gfx/map/map_object_data/building_locators.txt

```diff
@@ -1759,7 +1759,7 @@
 		}
 		{
 			id=294
-			position={ 3673.000000 0.000000 2232.000000 }
+			position={ 3671.783203 0.000000 2236.604004 }
 			rotation={ -0.000000 -0.000000 -0.000000 1.000000 }
```

Comment:
Don't forget to change provinces. May ask @ElMariuso for help since he may be interested.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/133263037-52d38340-9fec-4faf-939b-4319f1202791.png)

</details>

Reply:
Fixed in 3ca060b.